parser.def

```diff
@@ -0,0 +1,68 @@
+start
```

Comment:
Maybe it is better to also clone the compiler repository and take the def file committed there to avoid inconsistencies.